app/templates/views/api/index.html

```diff
@@ -39,9 +39,11 @@ <h2 class="heading-small">
         <p class="api-notifications-item__meta">
           When you send messages via the API they’ll appear here.
         </p>
+        {% if service.get_consistent_data_retention_period() %}
         <p class="api-notifications-item__meta">
-          Notify deletes messages after 7 days.
+          Notify deletes messages after {{ service.get_consistent_data_retention_period() }} days.
```

Comment:
You don’t need to pass in `service` here, `current_service` is available as a global in all Jinja templates
```suggestion
          Notify deletes messages after {{ current_service.get_consistent_data_retention_period() }} days.
```

Reply:
Good spot. Have changed and rebased this in.